<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>99</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>초음파</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pir_btn_on">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>99</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>불 켜기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pir_btn_off">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="spk_btn_on">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>30</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>스피커</string>
    </property>
   </widget>
   <widget class="QPushButton" name="light_btn_on">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="light_btn_off">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="motor_btn_on">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="motor_btn_off">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>모터</string>
    </property>
   </widget>
   <widget class="QPushButton" name="spk_btn_off">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>70</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>190</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>CAMERA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmr_btn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>410</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QLabel" name="photo_label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>250</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>picture</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
